src/taco-cli/cli/commands.json

```diff
@@ -399,6 +399,9 @@
             ,
             {
                 "name": "--target=TARGET", "description": "[CommandRunTargetDescription]"
+            },
+            {
+                "name":  "--list", "description": "[CommandRunListDescription]"
```

Comment:
The option is already there (--device and --emulate)

Cordova lays its help out in a different way to us, with the options grouped into "bags" such as the DEVICECLASS bag which is --device or --emulator, the PROD bag of --debug or --release, etc.